core-common/src/main/java/org/zalando/nakadi/service/subscription/zk/SubscriptionRebalancer.java

```diff
@@ -1,4 +1,4 @@
-package org.zalando.nakadi.service.subscription;
```

Comment:
Is it necessary to move it to a different package, or it's more of a style thing?

Reply:
This class is now used only from one place (not counting the class's test).
It was slightly relocated with this change in order to keep it package private (by moving it to the package where it's used).